testonly/hammer/record.go

```diff
@@ -28,48 +30,95 @@ const (
 	smrCount = 30
 )
 
-// smrStash provides thread-safe access to an ordered, bounded queue of previously
-// witnessed SMRs
-type smrStash struct {
-	mu sync.RWMutex
+// sharedState allows details of what has been written to and read from the Map
```

Comment:
simplify: "allows details ... to be shared between ..." -> "shares the details ... between ..."